File: Pemrograman1/Modul2/C/PRAK204-2310817310009-MuhammadFauzanAhsani.c
#include <stdio.h>

void main(){
    double radius, height, pi = 22.0 / 7.0;
    scanf("%lf %lf", &radius, &height);

    printf("Volume = %.2lf \n", radius*radius * pi * height);
    printf("Luas = %.2lf \n", 2 * ((pi * radius) * (radius + height)) );
    printf("Keliling = %.2lf \n", 2 * pi * radius );
}

File: Pemrograman1/Modul2/Python/PRAK204-2310817310009-MuhammadFauzanAhsani.py
radius, height = input().split()
radius, height = map(float, (radius, height))
pi = 22.00/7.00

print(f"Volume = {radius**2 * pi * height:.2f}")
print(f"Luas = {2 * ((pi * radius) * (radius + height)):.2f}")
print(f"Keliling = {2 * pi * radius:.2f}")